omahelsinki/settings.py

```diff
@@ -134,8 +137,17 @@
 STATICFILES_FINDERS = [
     'django.contrib.staticfiles.finders.FileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'compressor.finders.CompressorFinder',
```

Comment:
Did you remember to update `requirements.in` for `django_compressor` and `django-libsass`? After modifying the `.in` file, update the pinned versions by the `pip-compile -U` command.